src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -243,25 +243,46 @@ private String get_user_myworkspace_site_json(String userEid, String sessionId)
 		String errorMessage = "";
 		String requestUrl = "";
 
-		// get all sites that user is of Owner role
+		// First, get all sites that user is of Owner role
+		JSONArray sitesJSONArray = getSitesJSONArray(currentUserId, sessionId, allowedSiteTypes, Utils.ROLE_OWNER);
+		// Then append all sites that user is of Instructor role
+		sitesJSONArray.put(getSitesJSONArray(currentUserId, sessionId, allowedSiteTypes, Utils.ROLE_INSTRUCTOR));
```

Comment:
will there be any overlap of sites with instructor and owner roles? if yes, will that matter?

Reply:
Good idea! I checked on the database and this won't be a case:

SELECT count(*), USER_ID, REALM_KEY
FROM CTOOLS_USER.SAKAI_REALM_RL_GR
GROUP BY USER_ID, REALM_KEY
ORDER BY count(*) desc